Dreamscapes/Ldap/Core/Ldap.php

```diff
@@ -714,7 +719,7 @@ public function startTls()
      */
     public function unbind()
     {
-        ldap_unbind($this->resource);
+        @ldap_unbind($this->resource);
```

Comment:
Okay, that sounds logical... Calling `ldap_unbind()` on unbound resource seems to only cause a warning, so let's keep it as is - with silencer and without throwing anything.